rename.rb

```diff
@@ -32,7 +32,7 @@ def usage
 old_dirs = []
 Find.find('.') do |path|
   next unless File.file?(path)
-  next if path =~ /\.git/
+  next if /\.git/.match?(path)
```

Comment:
This is the same
```suggestion
  next if path.include?('.git')
```

But this should really use a different approach and that is to use `Find.prune` as noted in https://github.com/theforeman/foreman_plugin_template/pull/62#discussion_r1554074765